data/ecosystems/y/yield-eos.toml

```diff
@@ -0,0 +1,16 @@
+# Ecosystem Level Information
+title = "Yield+ on EOS"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/eosnetworkfoundation"]
```

Comment:
EOS Network Foundation org is already part of Antelope, which is a child eco of EOS